apps/fluent-tester/src/RNTester/RNTester.tsx

```diff
@@ -1,15 +1,91 @@
 'use strict';
```

Comment:
Do we know how this component gets used? If it doesn't need changes we can ignore it, or better yet if it isn't used, lets delete it.